examples/simple-app/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Heading, Layout } from 'virtuoso-design-system/dist/bundle.esm'
```

Comment:
just curious, is there no way to avoid the **dist/bundle.esm** part of the path? would it help to have a **dist/index.js**? Does `main` or `module` (in the top-level **package.json**) not handle this?

Reply:
Yea this is depending on a new release. I was running it locally and parcel had some issues... 🙄